app/src/main/java/com/example/foodjuju/MainActivity.kt

```diff
@@ -16,10 +16,13 @@ class MainActivity : AppCompatActivity() {
         val navView: BottomNavigationView = findViewById(R.id.nav_view)
 
         val navController = findNavController(R.id.nav_host_fragment)
-        // Passing each menu ID as a set of Ids because each
-        // menu should be considered as top level destinations.
-        val appBarConfiguration = AppBarConfiguration(setOf(
-            R.id.navigation_home, R.id.navigation_calendar, R.id.navigation_report))
+        /* Passing each menu ID as a set of Ids because each
```

Comment:
I'd save /* */ for true JavaDoc style comments, which are generally only above a class or method.